### PR TITLE
Fix welder making unwanted sounds

### LINKED ISSUE #1532: The welder still makes sounds if it's empty/unlit
1. Equip a unlit/empty welder
2. Try to weld a locker or something
3. You hear the weld sound => no

### PR DESCRIPTION
**Pull request by exp111**
Fixes #1532

### CODE REVIEW

File: Content.Server/GameObjects/Components/Interactable/WelderComponent.cs [hunk 1]
```diff
@@ -116,7 +116,13 @@ private bool TryWeld(float value, IEntity? user = null, bool silent = false)
             if (_solutionComponent == null)
                 return false;
 
-            return _solutionComponent.TryRemoveReagent("chem.WeldingFuel", ReagentUnit.New(value));
+            bool succeeded = _solutionComponent.TryRemoveReagent("chem.WeldingFuel", ReagentUnit.New(value));
+
+            if (succeeded)
+            {
+                PlaySoundCollection("Welder");
```
Comment (Zumorica) [inline Content.Server/GameObjects/Components/Interactable/WelderComponent.cs:123]: Maybe add a serialized data field for this instead of hardcoding it

Comment (exp111) [reply Content.Server/GameObjects/Components/Interactable/WelderComponent.cs:123]: I actually had this serialized, but then i saw that the suicide method also uses the normal Welder soundcollection

**Comment by exp111** [comment]
oh this is gonna be fucky with multitools

File: Content.Server/GameObjects/Components/Interactable/WelderComponent.cs (hunk 1 above)
Comment (Zumorica) [reply Content.Server/GameObjects/Components/Interactable/WelderComponent.cs:123]: Make both serialized then :^)

**Comment by Zumorica** [comment]
> oh this is gonna be fucky with multitools

Technically no multitools use welders since it's a special case.

**Comment by exp111** [comment]
> no multitools

yet

**Comment by Zumorica** [comment]
> yet

I will personally campaign against multitools with welders if a PR ever comes up (unless it special-cases it in which case it would be fine)

**Pull request merged**